sdks/typescript/client-typescript-core/README.md

```diff
@@ -15,33 +15,35 @@ with some other library if you use it.
 `eventsource` is of course the only option when using the Web, so that is what we expect developers to use there,
 but it can be replaced for backend or Mobile.
 
-== Mechanisms for use
+See [FeatureHub](https://featurehub.io) for more details.
+
+## Mechanisms for use
```

Comment:
can we add before this something about installation? npm i featurehub-eventsource-sdk ... this will also install "featurehub-repository" package